### ISSUE TITLE
Incompatible with infinimend-1.12-1.0.2

### ISSUE THREAD

**Issue by Kasia1403** [coolsimulations/Infinity-Water-Bucket#6]
[debug.log](https://github.com/coolsimulations/Infinity-Water-Bucket/files/8007996/debug.log)
[latest.log](https://github.com/coolsimulations/Infinity-Water-Bucket/files/8007997/latest.log)

**Comment by coolsimulations** [comment]
Sadly, there is not much I can really do about this. CoreMods on Forge are not recommended, which is what both Infinity Water Bucket and InfiniMend are, for this very reason.

We both try to edit base vanilla code through a form of ASM and when one mod changes the vanilla code, the other mod has issues because the original code has been changed from what is expected. On modern Forge & Fabric versions Mixins (what I use) is a solution to this problem so that two mods can both edit the same piece of code. However, InifiMend was created before this and uses direct ASM which is what is causing the issue.

In short, there is no possible fix for this. You'll have to pick which mod you want more. I'll mark this as an incompatible mod on CurseForge. Thanks.
